Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: setup
+setup:
+	python3 -m venv venv && source venv/bin/activate && pip install -r requirements.txt
+
+.PHONY: lint
+lint:
+	flake8
+
+.PHONY: typing
+typing:
+	mypy tips tests
+
+.PHONY: runserver
+runserver:
+	uvicorn tips.main:app --reload
+
+.PHONY: cov
+cov:
+	pytest --cov=tips --cov-report=term-missing
```

Comment:
Might want to add a percentage coverage here with `--cov-fail-under=MIN`, e.g. if you want to hit 80% you'd put 80 there.

Reply:
TIL, thanks, added that.

Reply:
black should take care of most of it but yes, it can be verbose. I have a curated list of things to ignore in pylint.

Reply:
Definitely want to adopt it because it looks at code quality as well, can you please share the things to ignore? Thanks!